source/installation/deployment-and-management.rst

```diff
@@ -14,50 +14,24 @@ MinIO is a software-defined high performance distributed object storage server.
 You can run MinIO on consumer or enterprise-grade hardware and a variety
 of operating systems and architectures.
 
-MinIO supports two deployment modes: :guilabel:`Standalone` and 
-:guilabel:`Distributed`:
-
-Standalone Deployments
-   A single MinIO server with a single storage volume or folder. Standalone deployments are best suited for evaluation and initial development of applications using MinIO for object storage, *or* for providing an S3 access layer to single storage volume. Standalone deployments do not provide access to the full set of MinIO's advanced S3 features and functionality - specifically those dependent on :ref:`Erasure Coding <minio-erasure-coding>`.
-
-Distributed Deployments
-   One or more MinIO servers with *at least* four total storage volumes across all servers. Distributed deployments are best for production environments and workloads and support all of MinIO's core and advanced S3 features and functionality.
-
-   MinIO recommends a baseline topology of 4 nodes with 4 drives each for production environments.
-
 .. _minio-installation-comparison:
 
-The following table compares the key functional differences between
-:guilabel:`Standalone` and :guilabel:`Distributed` MinIO deployments:
-
-.. list-table::
-   :header-rows: 1
-   :widths: 20 40 40 
-   :width: 100%
-
-   * - 
-     - :guilabel:`Standalone`
-     - :guilabel:`Distributed`
-
-   * - Site-to-Site Replication
-     - Client-Side via :mc:`mc mirror`
-     - :ref:`Server-Side Replication <minio-bucket-replication>`
+MinIO supports three deployment topologies: 
 
-   * - Versioning
-     - No
-     - :ref:`Object Versioning <minio-bucket-versioning>`
+Single-Node Single-Drive
+  A single MinIO server with a single storage volume or folder. 
+  |SNSD| deployments are best suited for evaluation and initial development of applications using MinIO for object storage.
+  This topology is also known as :guilabel:`Standalone Mode`.
```

Comment:
On line 12 of the SNSD standalone page, you say it was "previously known as".
Is it previously, or is it also?

Reply:
:thinking: I guess at this point it's previously - standalone / filesystem mode are somewhat meaningless with the XL backend features in place, and frankly it was always confusing because a SNMD is technically a standalone MinIO instance too, so drawing a distinction there was painful.